### PR TITLE
Prettier v2, prettier/pluging-pug

### PR DESCRIPTION
**Pull request by webkom-R2D2**
- [x] Always Trailing Comma
- [x] Always Parenthesis for array functions
- [x] Lint for `.pug` with 2-width spacing

### CODE REVIEW

File: app/models/user.js [hunk 1]
```diff
@@ -12,63 +12,63 @@ const userSchema = new Schema({
     index: true,
     required: true,
     unique: true,
-    match: /^\w{5,}$/
+    match: /^\w{5,}$/,
```
Comment (SmithPeder) [inline app/models/user.js:15]: wanne make sure this works

**Review by LudvigHz — APPROVED** [review]

File: client/services/cardKeyService.js [hunk 2]
```diff
@@ -27,18 +27,18 @@ const validate = (data, receivedChecksum) => {
...
-  '84': 'RESPONSE ERROR',
-  '82': 'READER TIMEOUT',
-  '90': 'CARD DOES NOT SUPPORT THIS COMMAND',
-  '8f': 'UNSUPPORTED CARD IN NFC WRITE MODE'
+  83: 'NO CARD',
+  87: 'UNKNOWN INTERNAL ERROR',
+  85: 'UNKNOWN COMMAND',
+  84: 'RESPONSE ERROR',
+  82: 'READER TIMEOUT',
+  90: 'CARD DOES NOT SUPPORT THIS COMMAND',
```
Comment (SmithPeder) [inline client/services/cardKeyService.js:35]: hmm

**Review by SmithPeder — APPROVED** [review]

**Comment by SmithPeder** [comment]
The `match` probably works, and there are a lot of tests for it. Not really sure about the cardKeyService suddenly changing 60% of the keys from string to int tho, and there are no tests for that 👎 Will have a look

**Comment by SmithPeder** [comment]
JS why are you like this?
```js
map['1'] = 'string one';
map[2] = 'number two';

console.log(map[1]);   // 'string one'
console.log(map['1']); // 'string one'
```

**Pull request merged**